tests/utils/GraphicalUnitTester.py

```diff
@@ -119,6 +120,11 @@ def _show_diff_helper(self, frame_data, expected_frame_data):
         plt.show()
         plt.savefig(f"{self.scene}.png")
 
+    @deprecated(
+        until="0.12",
+        replacement="frames_comparison",
+        message="The way scenes are tested has changed. Refer to the latest doc.",
+    )
```

Comment:
**(suggestion)**: Should probably also include a `since` argument, plus added some `v' to be consistent with the rest of the deprecations in the library.
```suggestion
    @deprecated(
        since="v0.11.0",
        until="v0.12.0",
        replacement="frames_comparison",
        message="The way scenes are tested has changed. Refer to the latest doc.",
    )
```